### PR TITLE
feat: add unsafe_reset_all command to remove all storage

### LINKED ISSUE #717: add unsafe_reset_all subcommand

### PR DESCRIPTION
**Pull request by jeremyletang**
close #717

### CODE REVIEW

File: cmd/data-node/unsafe_reset_all.go [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+		log.Info("Removed storage state", logging.String("path", storagePath))
+	}
+	return nil
+}
+
+func deleteAll(log *logging.Logger, dir string) error {
+	d, err := os.Open(dir)
+	if err != nil {
+		return err
+	}
```
Comment (daniel1302) [inline cmd/data-node/unsafe_reset_all.go:52]:
```suggestion
	}
	defer d.Close()
```
But here program anyway is closed shortly so not required.

**Review by daniel1302 — APPROVED** [review]
Looks good. 

Tested also locally:

```
➜  vegacapsule git:(vegacapsule-128/unsafe-reset-all) ✗ data-node unsafe_reset_all --home /Users/daniel/.vegacapsule/testnet/data/node1
2022-06-23T04:45:39.338-0500    INFO    data-node/unsafe_reset_all.go:63        Removed file    {"path": "/Users/daniel/.vegacapsule/testnet/data/node1/state/data-node/storage/trades"}
2022-06-23T04:45:39.358-0500    INFO    data-node/unsafe_reset_all.go:63        Removed file    {"path": "/Users/daniel/.vegacapsule/testnet/data/node1/state/data-node/storage/checkpoints"}
2022-06-23T04:45:39.358-0500    INFO    data-node/unsafe_reset_all.go:63        Removed file    {"path": "/Users/daniel/.vegacapsule/testnet/data/node1/state/data-node/storage/candles"}
2022-06-23T04:45:39.377-0500    INFO    data-node/unsafe_reset_all.go:63        Removed file    {"path": "/Users/daniel/.vegacapsule/testnet/data/node1/state/data-node/storage/accounts"}
2022-06-23T04:45:39.377-0500    INFO    data-node/unsafe_reset_all.go:63        Removed file    {"path": "/Users/daniel/.vegacapsule/testnet/data/node1/state/data-node/storage/markets"}
2022-06-23T04:45:39.377-0500    INFO    data-node/unsafe_reset_all.go:63        Removed file    {"path": "/Users/daniel/.vegacapsule/testnet/data/node1/state/data-node/storage/orders"}
2022-06-23T04:45:39.377-0500    INFO    data-node/unsafe_reset_all.go:63        Removed file    {"path": "/Users/daniel/.vegacapsule/testnet/data/node1/state/data-node/storage/chain_info"}
2022-06-23T04:45:39.377-0500    INFO    data-node/unsafe_reset_all.go:43        Removed storage state   {"path": "/Users/daniel/.vegacapsule/testnet/data/node1/state/data-node/storage"}
```

File: cmd/data-node/unsafe_reset_all.go (hunk 1 above)
Comment (jeremyletang) [reply cmd/data-node/unsafe_reset_all.go:52]: it is definitely required, that was a mistaken of mine

**Review by pscott31 — APPROVED** [review]

**Comment by daniel1302** [comment]
Jenkins is failing with the below error: https://jenkins.ops.vega.xyz/job/data-node/job/PR-721/6/console

```
ERROR: Pull request 721 : Not mergeable at e945398e9d1ca3c1cab92605b693b4ee35780869+14ef089631c9244d8951f4794fec2f1dd4355ca7 (NOT_MERGEABLE)

GitHub has been notified of this commit’s build result

Finished: FAILURE
```

**Comment by jeremyletang** [comment]
Not necessary for now anymore as we do't have any state apart from in psql!

**Pull request closed without merging**